core-tests/src/test/java/org/mule/runtime/core/DefaultMuleEventTestCase.java

```diff
@@ -51,7 +52,7 @@
 
   @Before
   public void before() throws Exception {
-    flow = getTestFlow();
+    flow = getTestFlow(muleContext);
```

Comment:
Is it preferred to use static method in MuleTestUtils over method in AbstractMuleContextTestCase that does not require MuleContext? The later seems simple..